vnext/src/Libraries/AppTheme/AppTheme.ts

```diff
@@ -10,7 +10,7 @@ import {IHighContrastColors} from './AppThemeTypes';
 
 class AppThemeModule extends MissingNativeEventEmitterShim {
   public isAvailable = false;
-  public currentTheme = '';
+  public currentTheme: 'light' | 'dark' = 'light';
```

Comment:
This should probably be defined as a type, so you don't keep repeating it everywhere.

Reply:
sure, I meant to do that but wasn't finding a good name for the type.